### PR TITLE
Fix deps

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Review by mandyvenables — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,8 +25,6 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-account-http",
   "dependencies": {
-    "@bedrock/https-agent": "^4.0.0",
-    "@bedrock/turnstile": "^1.0.0",
     "@digitalbazaar/http-client": "^4.0.0",
```
Comment (JSAssassin) [inline package.json:28]: Looks like this is not being used too and can be removed?

File: package.json [hunk 2]
```diff
@@ -38,6 +36,7 @@
     "@bedrock/core": "^6.0.0",
     "@bedrock/express": "^8.0.0",
     "@bedrock/passport": "^11.0.0",
+    "@bedrock/turnstile": "^1.0.1",
```
Comment (JSAssassin) [inline package.json:39]: test is currently failing locally `@bedrock/turnstile` needs to be added to the test deps.

**Review by dlongley — APPROVED** [review]
Approving with @JSAssassin's suggestions.

**Comment by mattcollier** [comment]
At this point, CI is failing due to node.js 20.8.1 / fetch issues.  Tests pass locally with Node.js 20.80.  Merging.

**Pull request merged**